pattern-1/README.md

```diff
@@ -54,7 +57,7 @@ Nginx Ingress cluster IP and try navigating to `https://wso2is-pattern1/carbon`
 your favorite browser.
 
 ##### 7. Scale up using `kubectl scale`:
-Default deployment runs two replicas (or pods) of WSO2 Identity server. To scale this deployment into <br>
+Default deployment runs one replica (or pod) of WSO2 Identity server. To scale this deployment into <br>
 any `<n>` number of container replicas, upon your requirement, simply run following kubectl 
 command on the terminal. Assuming your current working directory is `KUBERNETES_HOME/pattern-1` 
```

Comment:
Changing the no of replicas will not comply with IS deployment pattern-1

Reply:
@malithie I believe this approach has been taken due to the database entry duplication exception which was experienced during Identity Server startup (when creating two replicas in parallel).

This was experienced earlier when building BOSH Releases for Identity Server deployment patterns [1]. It wasn't possible to start both instances of Identity Server in parallel.

Please correct me if I am wrong, but I think this issue has to be fixed within the product. Plus, I think starting with a single replica and scaling it up to comply with pattern 1 is a correct approach.

I think the above README documentation needs to clearly state the real reason for scaling (to comply with the pattern, which is a mandatory step). Currently, this step looks to me like an optional step.

[1]: [BOSH Releases for Identity Server deployment patterns](https://github.com/wso2/pivotal-cf-is)